wp-content/themes/cpnet-developer/reference/template-changelog.php

```diff
@@ -19,7 +19,13 @@
 			<caption class="screen-reader-text"><?php _e( 'Changelog', 'wporg' ); ?></caption>
 			<thead>
 				<tr>
-					<th class="changelog-version"><?php _e( 'Version', 'wporg' ); ?></th>
+					<th class="changelog-version"><?php _e( 'Version', 'wporg' ); ?><span class="btn btn-primary tooltip"><span>?</span>
+	    				<div class="right">
+	        				<span><?php _e( 'Versions with <code>WP-</code> prefix describe changes made in WordPress prior to ClassicPress fork.', 'wporg' ); ?></span>
```

Comment:
These changes are not necessarily "prior to ClassicPress fork" as stated. I suggest a very brief explanation here that links to a new doc page about internal versions:

```suggestion
	        				<span><?php _e( 'Versions with <code>WP-</code> prefix describe changes made in WordPress. <a href="/developer-guides/cp-wp-versions/">More info</a>', 'wporg' ); ?></span>
```

Reply:
Makes sense.
I created a draft doc here https://docs.classicpress.net/wp-admin/post.php?post=17846&action=edit, can you check if that is adequate?
You have to login to access it. I chose a different title (Versioning Policy), instead of "cp-wp-versions" for clarity.